torch/nn/modules/pooling.py

```diff
@@ -524,6 +524,9 @@ class AvgPool1d(_AvgPoolNd):
               L_{out} = \left\lfloor \frac{L_{in} +
               2 \times \text{padding} - \text{kernel\_size}}{\text{stride}} + 1\right\rfloor
 
+          If ``ceil_mode`` is True and :math:`(L_{out} - 1) \times \text{stride} \geq L_{in} + \text{padding}`,
+          we skip the last window, resulting in :math:`L_{out}` being reduced by one.
```

Comment:
nit: I think you can add a short sentence here like "Per the note above,"

```suggestion
          Per the note above, if ``ceil_mode`` is True and :math:`(L_{out} - 1) \times \text{stride} \geq L_{in} + \text{padding}`,
          we skip the last window as it would start in the right padded region, resulting in :math:`L_{out}` being reduced by one.
```

Reply:
Indeed, much better like this.